src/Flow.jl

```diff
@@ -128,11 +144,13 @@ and the `AbstractPoisson` pressure solver to project the velocity onto an incomp
     a.u⁰ .= a.u; scale_u!(a,0)
     # predictor u → u'
     conv_diff!(a.f,a.u⁰,a.σ,ν=a.ν,perdir=a.perdir)
+    applyBodyForce!(a.f,time(a),a.g)
```

Comment:
Would it be more efficient to have a conditional here so that we avoid adding `0` when no body-force function is passed? Could you please check this @TzuYaoHuang?

Reply:
That is a good idea. I think I can add a flag in `Flow` struct and do sth generative function like in Periodic BC